### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by felixlee0530**
Korean translation file (gui.json) added.

### CODE REVIEW

File: assets/romfs/i18n/ko/gui.json [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+    "BANK_CONFIRM_CLEAR": "해당 박스를 지우겠습니까?",
+    "BANK_CONFIRM_DUMP": "선택한 포켓몬을 덤프하겠습니까?",
+    "BANK_CONFIRM_RELEASE": "선택한 포켓몬을 놓아주겠습니까?",
+    "BANK_CONVERT": "저장소 변환 중...",
+    "BANK_CORRUPT": "손상된 저장소 데이터",
+    "BANK_CREATE": "저장소 생성 중...",
+    "BANK_FAILED_EXIT": "포켓몬을 집고 있을 때에는 나갈 수 없습니다!",
+    "BANK_LOAD": "저장소를 불러오는 중...",
+    "BANK_NAME_ERROR": "박스 이름을 저장할 수 없습니다!",
+    "BANK_OPEN_FAILED": "Impossibile avviare lo scaricamento!",
```
Comment (piepie62) [inline assets/romfs/i18n/ko/gui.json]: Could you translate this last one? In English it's "Storage open failed!"

**Comment by BernardoGiordano** [comment]
@felixlee0530 any update on this?

**Comment by felixlee0530** [comment]
Oops.. I was so busy recently..
I fixed the Spanish part to the one you suggested.
@BernardoGiordano

**Comment by BernardoGiordano** [comment]
Thank you

**Pull request merged**